src/Avalonia.Input/KeyboardNavigation.cs

```diff
@@ -47,7 +47,7 @@ public static class KeyboardNavigation
         /// </summary>
         /// <param name="element">The container.</param>
         /// <returns>The <see cref="KeyboardNavigationMode"/> for the container.</returns>
-        public static KeyboardNavigationMode GetTabNavigation(InputElement element)
+        public static KeyboardNavigationMode GetTabNavigation(IAvaloniaObject element)
```

Comment:
Why not IInputElement?

Reply:
IInputElement does not extend IAvaloniaObject making it impossible to query or set the attached properties.
But your comment made me check again and replace some uses of this function with the interface-property on IInputElement.
The only remaining uses will be from the implementations of the IInputElement#TabNavigation property.

Reply:
> IInputElement does not extend IAvaloniaObject

I mean, it can extend it, is't it? It doesn't require dependency on IVisual.